fuel-vm/src/tests/blockchain.rs

```diff
@@ -299,21 +462,23 @@ fn load_external_contract_code() {
 
     let consensus_params = ConsensusParameters::standard();
 
-    let tx_create_target = TransactionBuilder::create(program.clone(), salt, vec![])
-        .gas_price(gas_price)
-        .gas_limit(gas_limit)
-        .maturity(maturity)
-        .add_random_fee_input()
-        .add_output(output0)
-        .finalize()
-        .into_checked(height, &consensus_params)
-        .expect("failed to check tx");
+    let tx_create_target =
+        TransactionBuilder::create(target_contract_witness.clone(), salt, vec![])
+            .gas_price(gas_price)
+            .gas_limit(gas_limit)
+            .maturity(maturity)
+            .add_random_fee_input()
+            .add_output(output0)
+            .finalize()
+            .into_checked(height, &consensus_params)
+            .expect("failed to check tx");
 
     client.deploy(tx_create_target);
 
     // Then deploy another contract that attempts to read the first one
     let reg_a = 0x20;
-    let reg_b = 0x21;
+    let reg_c = 0x21;
+    let reg_b = 0x22;
```

Comment:
```suggestion
    let reg_b = 0x21;
    let reg_c = 0x22;
```

Maybe you want to have something like this=D